src/test/test_ceph_daemon.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nosetests
```

Comment:
it is surprising to see nosetests being added to more places in the codebase. That project is no longer maintained (it hasn't received updates in a while). I don't think using it is needed at all, and pytest would work fine.